index.js

```diff
@@ -30,4 +30,220 @@ closeHam.addEventListener('click', closeHamburger);
+              <li><a href="#" class="boot">${firstProject.tech3}</a></li>
+              <li><a href="#" class="ruby">${firstProject.tech4}</a></li>
+      </ul>
+      <button class="pro1-text hover-effect" onclick='firePopUp()'>${firstProject.button}</button>
+  </div>`;
+}
+
+/* Adding an event listener to the window object. The event listener
+is listening for the load event. When the load event is triggered,
+the firstProjectFunction is called.
+*/
+window.addEventListener('load', firstProjectFunction);
+
+/* The `const remainingCards = document.querySelector('.cards-frame');` is selecting the
+`cards-frame` class from the DOM. */
+const remainingCards = document.querySelector('.cards-frame');
+
+/* Creating an array of objects for the remaining project cards. */
+const remainingProjects = [
+  {
+    id: 2,
+    title: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    tech1: 'css',
+    tech2: 'html',
+    tech3: 'Ruby',
+    button: 'See Project',
+  },
+  {
+    id: 3,
+    title: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    tech1: 'css',
+    tech2: 'html',
+    tech3: 'Ruby',
+    button: 'See Project',
+  },
+  {
+    id: 4,
+    title: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    tech1: 'css',
+    tech2: 'html',
+    tech3: 'Ruby',
+    button: 'See Project',
+  },
+  {
+    id: 5,
+    title: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    tech1: 'css',
+    tech2: 'html',
+    tech3: 'Ruby',
+    button: 'See Project',
+  },
+  {
+    id: 6,
+    title: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    tech1: 'css',
+    tech2: 'html',
+    tech3: 'Ruby',
+    button: 'See Project',
+  },
+  {
+    id: 7,
+    title: 'Profesional Art Printing Data',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    tech1: 'css',
+    tech2: 'html',
+    tech3: 'Ruby',
+    button: 'See Project',
+  },
+
+];
+
+/*
+ The function loops through the remainingProjects array and adds the project's title, description,
+ tech1, tech2, tech3, and button to the remainingCards div
+ */
+function remainingProjectFunc() {
+  remainingProjects.forEach((project) => {
+    remainingCards.innerHTML += `<div class="all-cards">            
+       <div class="no-btn">
+          <h2>${project.title}</h2>
```

Comment:
- [ ] I think it would be great if you add more font weight to the title to let it match the figma design as much as possible ✅💯


- Your design
![image](https://user-images.githubusercontent.com/58553711/225185971-ef396175-47e1-469a-afa9-bd9097693026.png)


- Figma
![image](https://user-images.githubusercontent.com/58553711/225186016-1c654e3b-08b2-43ad-991c-d8e2964ddf63.png)